test/report.test.ts

```diff
@@ -65,6 +65,35 @@ describe('report', () => {
                     .then(null, (error: Error) => expect(error.message).toEqual('Bad Request'));
             });
 
+            it('should return restricted error', () => {
+                const finishedTask = {
+                    status: 400,
+                    // tslint:disable-next-line:max-line-length
+                    body: '{"error":{"message":"During export we\'ve detected user error: Export to required format is not allowed for data flagged as restricted."}}'
+                };
+                const runningTask = { status: 202, uri: createdReport };
+
+                fetchMock.mock(
+                    projectUri,
+                    finishedTask
+                );
+
+                mockPollingRequest(createdReport, runningTask, finishedTask);
+
+                const exportConfig: IExportConfig = {
+                    title: 'title',
+                    format: 'xlsx',
+                    mergeHeaders: false
+                };
+
+                return mockedReportModule()
+                    .exportResult(projectId, executionResult, exportConfig, { pollStep: 1 })
+                    .then(null, (error: Error) => expect(error.message).toEqual(
```

Comment:
I think we can use `.toBe` to assert `string`. AFAIK, `toEqual` is used to assert `object`

Reply:
updated